src/Symfony/Component/HttpFoundation/Tests/ResponseTest.php

```diff
@@ -223,7 +255,7 @@ public function testIsNotModifiedLastModifiedAndEtag()
 
         $response->headers->set('ETag', $etag);
         $response->headers->set('Last-Modified', $after);
-        $this->assertFalse($response->isNotModified($request));
+        $this->assertTrue($response->isNotModified($request));
```

Comment:
Why did this change?

Reply:
The RFC (https://tools.ietf.org/html/rfc7232#section-3.3) states:

>    A recipient MUST ignore If-Modified-Since if the request contains an
>    If-None-Match header field; the condition in If-None-Match is
>    considered to be a more accurate replacement for the condition in
>    If-Modified-Since, and the two are only combined for the sake of
>    interoperating with older intermediaries that might not implement
>    If-None-Match.

Reply:
You are absolutely right. I should have read the code comments. 🙈